api/admin/controller/report.py

```diff
@@ -31,21 +28,17 @@ def generate_inventory_report(self) -> Response | ProblemDetail:
             assert admin.id
             assert library.id
 
-            data: InventoryReportTaskData = InventoryReportTaskData(
-                admin_email=admin.email, admin_id=admin.id, library_id=library.id
-            )
-            task, is_new = queue_task(
-                self._db, task_type=DeferredTaskType.INVENTORY_REPORT, data=asdict(data)
+            generate_inventory_and_hold_reports.delay(
+                email_address=admin.email, library_id=library.id
             )
 
             msg = (
-                f"An inventory report request was {'already' if not is_new else ''} received at {task.created}. "
-                f"When processing is complete, the report will be sent to {admin.email}."
+                f"An inventory and hold report request was received. Report processing can take a few minutes to "
+                f"finish depending on current server load. The completed reports will be sent to {admin.email}."
             )
 
-            self.log.info(msg + f" {task}")
-            http_status = HTTPStatus.ACCEPTED if is_new else HTTPStatus.CONFLICT
-            return Response(json.dumps(dict(message=msg)), http_status)
+            self.log.info(msg)
```

Comment:
Minor: If you wanted to log the task id, you could get the task from the `.delay` call and log its `.id` here. I only mention that because we were logging the task ID before. Might not be necessary though.